packages/app/pages/index.tsx

```diff
@@ -1,64 +1,265 @@
 import ModelIcon from '../components/model-icon'
 import PageTitle from '../components/page-title'
 import UnderMaintenance from '../components/under-maintenance'
 import { getModelsWithDocumentCount } from '../models/service'
 import type { ModelCategory } from '../models/types'
-import { sortModels } from '../utils/sort'
 import styles from './dashboard.module.css'
+import type { DocumentComment } from '../comments/types'
 
 type DashboardPageProps = {
     modelCategories: ModelCategory[]
+    unresolvedUserComments: DocumentComment[]
+    userRecentDocuments: Document[]
 }
+const listFormatter = new Intl.ListFormat('en-US', {
+    style: 'long',
+    type: 'conjunction',
+})
+const timeFormatter = new Intl.DateTimeFormat('en-US', { dateStyle: 'medium' })
+
+const DashboardPage = ({
+    modelCategories,
+    unresolvedUserComments,
+    userRecentDocuments,
+}: DashboardPageProps) => {
+    const userIconMap = useMemo(
+        () =>
+            modelCategories.reduce((accumulator, current) => {
+                current.models.forEach(model => {
+                    accumulator[model.name] = model.icon
+                })
+
+                return accumulator
+            }, {}),
+        [modelCategories]
+    )
+    const userDocumentStates = useMemo(() => {
+        const uniqueDocumentStates = new Set<string>()
+
+        for (const document of userRecentDocuments) {
+            uniqueDocumentStates.add(document['x-meditor'].state)
+        }
+
+        return Array.from(uniqueDocumentStates).sort()
+    }, [userRecentDocuments])
+
+    const [filterModelBy, setFilterModelBy] = useState<'hasRoles' | 'none'>(
+        'hasRoles'
+    )
+    const [filterDocumentBy, setFilterDocumentBy] = useState<string>(
+        userDocumentStates.find(string => string === 'Under Review') ||
+            userDocumentStates[0]
+    )
 
-const DashboardPage = ({ modelCategories }: DashboardPageProps) => {
     return (
-        <div>
+        <main className={styles.main}>
             <PageTitle title="" />
 
-            {(!modelCategories || modelCategories.length < 0) && <UnderMaintenance />}
-
-            {modelCategories?.map(category => (
-                <div key={category.name} className={styles.category}>
-                    <h3>{category.name}</h3>
-
-                    <div className={styles.models}>
-                        {category.models
-                            .sort((a, b) => a.name.localeCompare(b.name))
-                            .map(model => (
-                                <div key={model.name} className={styles.model}>
-                                    <Link href={`/${model.name}`}>
-                                        <Button
-                                            variant="light"
-                                            className="dashboard-model"
+            {!modelCategories.length && <UnderMaintenance />}
+
+            <article className={`${styles.card} rounded shadow-sm`}>
```

Comment:
There are three fairly similar `<article>` cards on this dashboard page now. I know that I'm bordering on needing to refactor this, but I often find that early abstractions need more refactoring later than if I would have just waited a bit to see a few different use cases. If it doesn't bother you all too much, I'd like to wait until we get a few different cards / modules on the dashboard before figuring out the right abstraction—right now they're all pretty similar in that they display a list of links.